deploy/openshift/templates/che-server-template.yaml

```diff
@@ -292,6 +306,34 @@ parameters:
   displayName: Plugin sidecar default memory limit
   description: Plugin sidecar default memory limit in megabytes
   value: '128'
+- name: CHE_TRACING_ENABLED
+  displayName: Eclipse Che tracing
+  description: Enable or disable tracing in Eclipse Che
+  value: 'false'
+- name: JAEGER_SERVICE_NAME
+  displayName: Jaeger service name
+  description: Jager service name
+  value: 'che'
+- name: JAEGER_AGENT_HOST
+  displayName: Jaeger agent host
+  description: The hostname for communicating with jager agent via UDP
+  value: 'jaeger-agent'
+- name: JAEGER_SAMPLER_TYPE
+  displayName: Jageer sampler type
+  description: Type of jager sampler can be - constant or probabilistic or ratelimiting or remote
+  value: 'const'
+- name: JAEGER_SAMPLER_PARAM
+  displayName: Jaeger sampler param
+  description: The sampler parameter (number)
+  value: '1'
+- name: JAEGER_REPORTER_MAX_QUEUE_SIZE
+  displayName: Jaeger reporter max queue size
+  description: The reporter's maximum queue size
+  value: '10000'
+- name: JAEGER_SAMPLER_MANAGER_HOST_PORT
+  displayName: Jaeger sampler manager host and port
+  description: The host name and port when using the remote controlled sampler
+  value: 'jaeger:5778'
```

Comment:
Maybe it makes sense to make it optional parameter without default value, and set it only when `JAEGER` is deployed?